plugins/jms-service-activemq/src/main/java/com/thinkbiganalytics/jms/activemq/ActiveMqConfig.java

```diff
@@ -44,7 +44,6 @@
 @Profile("jms-activemq")
 @Configuration
 @PropertySources({
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the 'PropertySources' wrapper from this annotation group [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1710)